web/webpack.dev-assets.json

```diff
@@ -15,12 +15,5 @@
         "./src/reload_state.ts",
         "./src/channel.ts"
     ],
-    "dev-buttons": [
-        "./src/bundles/portico.ts",
-        "./src/portico/design-testing.ts",
-        "./styles/portico/dev-buttons.css",
-        "./styles/app_variables.css",
-        "./styles/app_components.css",
-        "./styles/buttons.css"
-    ]
+    "dev-testing": ["./src/bundles/design-testing.ts"]
```

Comment:
`design-components-playground` might be a better name to be working towards for this part of the codebase.

Reply:
https://chat.zulip.org/#narrow/channel/6-frontend/topic/design.20components.20sandbox.20naming/near/2029695 for deciding what to use here.

Reply:
Addressed with the "showroom" naming convention following the latest discussions at CZO.